web/wp-content/themes/dctx/archive.php

```diff
@@ -1,18 +1,24 @@
-<?php get_header(); ?>
+<?php 
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).